libmscore/spanner.h

```diff
@@ -100,6 +100,8 @@ class SpannerSegment : public Element {
 
       QByteArray mimeData(const QPointF& dragOffset) const override;
 
+      virtual void spatiumChanged(qreal /*oldValue*/, qreal /*newValue*/) override;
```

Comment:
There is no need to have parameters names commented out here. Also `virtual` keyword can be omitted as we have `override`. So I would suggest declaring this function like
```
void spatiumChanged(qreal oldValue, qreal newValue) override;
```

Reply:
Well, @mattmcclinch told me the `virtual` has been added traditionally in MuseScore code...

Reply:
Maybe that should be defined in coding style explicitly. Anyway, yes, there are many places where `virtual` is used in overrides, so we can leave it as it is for now.